.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Thank you for your interest in ESLint! Please fill out the appropriate template for your issue below, as this will help us greatly in addressing your issue:
+
+1. Bug Reports
+2. New Rules
+3. Change Requests
+4. For any other questions, please stop by our chatroom: https://gitter.im/eslint/eslint
+
+Note that issues submitted with incomplete information may be closed without comment.
```

Comment:
I tried to soften it from the [original suggestion](https://github.com/eslint/eslint/issues/5313), which was "If you're reporting a bug, please continue on. Note that bug reports without all questions answered will be closed without comment."